### PR TITLE
Respect Astropy IERS URL configuration

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

File: astroplan/utils.py [hunk 1]
```diff
@@ -63,20 +64,21 @@ def IERS_A_in_cache():
...
-    except (IOError, OSError) as e:
-        msg = 'Remote data cache could not be accessed due to '
-        estr = '' if len(e.args) < 1 else (': ' + str(e))
-        warnings.warn(CacheMissingWarning(msg + e.__class__.__name__ + estr))
-        return False
-    with _open_shelve(urlmapfn, True) as url2hash:
-        # TODO: try to figure out how to test this in the unicode case
-        if str(url_key) in url2hash:
-            return True
+    urls = (conf.iers_auto_url, conf.iers_auto_url_mirror)
```
[the remark this answers is not in the document]
Comment (lpsinger) [reply astroplan/utils.py]: Oops. It should be `urls = (iers.conf.iers_auto_url, iers.conf.iers_auto_url_mirror)`.

File: astroplan/utils.py [hunk 2]
```diff
@@ -119,11 +121,20 @@ def download_IERS_A(show_progress=True):
     show_progress : bool
         `True` shows a progress bar during the download.
     """
+    urls = (conf.iers_auto_url, conf.iers_auto_url_mirror)
```
[the remark this answers is not in the document]
Comment (lpsinger) [reply astroplan/utils.py]: Fixed.

File: astroplan/utils.py [hunk 3]
```diff
@@ -119,11 +121,20 @@ def download_IERS_A(show_progress=True):
...
+
     if IERS_A_in_cache():
-        clear_download_cache(iers.IERS_A_URL)
+        for url in urls:
+            clear_download_cache(url)
+
+    for i, url in enumerate(urls):
+        try:
+            local_iers_a_path = download_file(iers.IERS_A_URL, cache=True,
```
[the remark this answers is not in the document]
Comment (wtgee) [reply astroplan/utils.py]: Why doesn't this change?

**Comment by wtgee** [comment]
FWIW, I'm using this patch in my fork because I don't have any other work around for our live telescopes. It's working great. Would be nice to get it merged. @bmorris3 maybe you can kick off the tests again?

**Comment by wtgee** [comment]
@bmorris3 @lpsinger anything I can do to help get some movement on this? I have to do a lot of manual restarting of our CI tests just because the IERS download has failed and this will help fix that. Thanks!

**Comment by wtgee** [comment]
@bmorris3 also, see comment in https://github.com/astropy/astropy/issues/8981#issuecomment-544735592.

The default IERS servers are going down in two days and will be down for 6 months.

**Comment by bmorris3** [comment]
@wtgee Sorry for the delay. These are scary times indeed! Tests restarted, will merge pending tests pass.

**Comment by bmorris3** [comment]
This build is failing with the following peculiar error: 
```
sh: 0: Can't open /etc/init.d/xvfb
```
I'm going to see if I can find the source...

**Comment by wtgee** [comment]
> [This build](https://travis-ci.org/astropy/astroplan/jobs/574617989#L734) is failing with the following peculiar error:
> 
> ```
> sh: 0: Can't open /etc/init.d/xvfb
> ```
> 
> I'm going to see if I can find the source...

Not sure why it's giving an error with these changes, but it's part of the matplotlib testing:

https://github.com/astropy/astroplan/blob/4693ed89fc52237e261a0ece194ee32940c3c380/.travis.yml#L91-L96

**Comment by bsipocz** [comment]
Yes, that whole section is very obsolete and can be removed. 

However it doesn't seem to be true that you don't do mpl tests, there is at least one in `test_sky.py`, even though it may be a dummy one.

Anyway, the current way to set this up is to use set `SETUP_XVFB=True` in the global variable part of the travis config.

**Pull request merged**